packages/sequence-diagram/src/mergeWindow.ts

```diff
@@ -50,7 +50,7 @@ const buildLoop = (merge: Merge): Loop => {
   return {
     nodeType: NodeType.Loop,
     count: merge.length,
-    digest: 'loop',
+    digest: ['loop', merge.length].join(':'),
```

Comment:
If we report the number of loop iterations as a change to the digest, then this is going to make the diagram digest pretty unstable. 

We may need to think about this as a metric that we track rather than as a structural change.